demonstrations/tutorial_How_to_simulate_quantum_circuits_with_lightning_tensor.py

```diff
@@ -0,0 +1,170 @@
+simulator, which can harness the computational power of Nvidia GPUs. This deivce is built on top of the C/C++ APIs offered by the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`__ library, a component of the `Nvidia cuQuantum SDK <https://docs.nvidia.com/cuda/cuquantum/latest/index.html>`__.
+The ``lightning.tensor`` device is designed to simulate large-scale quantum circuits efficiently. It can handle circuits with 30+ qubits with a manageable degree of entanglement within only a single GPU. Note that ``lightning.tensor`` is still under active development, and further improvements, 
+new features, and additional tutorials are expected in future releases.
+
+In this tutorial, we demonstrate how to simulate quantum circuits using the `lightning.tensor` device in PennyLane. The :class:`~pennylane_lightning.lightning_tensor.LightningTensor` device, similar to the :class:`~pennylane.devices.default_tensor.DefaultTensor` device, supports both Matrix Product State (MPS) and Exact Tensor 
+Network (ExactTN) representations of quantum states. For more information on the MPS and TN methods, we refer to the `Matrix Product States and Tensor Networks </tutorial_How_to_simulate_quantum_circuits_with_tensor_networks>`.
+
+Check the latest functionality in the :class:`documentation <.pennylane_lightning.lightning_tensor.LightningTensor>` or pick among other `PennyLane devices <https://pennylane.ai/plugins/#built-in-devices>`__ for your project.
+
+.. figure:: ../_static/demonstration_assets/how_to_simulate_quantum_circuits_with_tensor_networks/TN_MPS.gif
+    :align: center
+    :width: 90%
+
+"""
+
+######################################################################
+# Simulating a quantum circuit with the MPS method
+# ------------------------------------------------
+#
+# Let's start by showing how to simulate a quantum circuit using the MPS method. Generally, the MPS method can be used to simulate quantum circuits that are too large for state-vector simulations or too deep (higher degree of entanglement) for the exact TN contract method.
+# The circuit we show here is a simple example to demonstrate the features of a MPS method mentioned above.
+#
+
+import pennylane as qml
+import numpy as np
+
+# Define the keyword arguments for the MPS method
+kwargs_mps = {
+    # Maximum bond dimension of the MPS
+    "max_bond_dim": 128,
+    # Cutoff parameter for the singular value decomposition
+    "cutoff": np.finfo(np.complex128).eps,
+}
+
+# Parameters of the quantum circuit
+theta = 0.5
+phi = 0.1
+depth = 10
+n = 1011
+num_qubits = 100
+
+# Instantiate the device with the MPS method and the specified kwargs
+dev = qml.device("lightning.tensor", wires=num_qubits, method="mps", **kwargs_mps)
+
+
+# Define the quantum circuit
+@qml.qnode(dev)
+def circuit(theta, phi, n, num_qubits):
+    for _ in range(1, depth - 1):
+        for qubit in range(num_qubits - 1):
```

Comment:
I do not know exactly what does this code but a way to fix it is the following:
```suggestion
        for qubit in range(num_qubits - 4):
```
Doing this change and running the code I get the following output:
```
Number of qubits: 100
Result: 0.05707759499207281
Execution time: 5365.3981 seconds
```